Gopkg.lock

```diff
@@ -9,15 +9,9 @@
 
 [[projects]]
   branch = "master"
-  name = "github.com/Azure/go-ansiterm"
```

Comment:
But I have split this PR in two separate commits on purpose, so that we don't hide this in only one commit.